section2/exercises/if_statements.rb

```diff
@@ -61,5 +63,11 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
+cups_of_flour = 3
 has_sauce = true
+
+if cups_of_flour >= 2 && has_sauce == true
```

Comment:
nice solutions throughout this file; your code is easy to read because you follow Ruby conventions - so keep that up 💯